server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -104,10 +107,21 @@ public TokenService(ProjectManager projectManager, CommandExecutor executor,
     @ResponseConverter(CreateApiResponseConverter.class)
     public CompletableFuture<HttpResult<Token>> createToken(@Param String appId,
                                                             @Param boolean isAdmin,
-                                                            Author author, User loginUser) {
+                                                            Author author, User loginUser,
+                                                            @Nullable String secret) {
```

Comment:
How secret is extracted from the request?
Could you add a test case for that?

Reply:
Oops...